techmeet/apps/frontend/apps.py

```diff
@@ -0,0 +1,8 @@
+"""Frontend Apps."""
```

Comment:
You can remove this file. We're not importing it in __init__.py and not defining the full app path in the settings so it's not actually being used.

Reply:
Removed in https://github.com/keeners/techmeet/pull/11/commits/09ab9c85f12571b54d5ec4f790afe24f3ed1ff60